web/css/main.css

```
@@ -2,6 +2,68 @@
 @tailwind components;
 @tailwind utilities;
 
+@font-face {
+    font-family: 'Erode';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Erode" (font-family-name-quotes)

(font-family-name-quotes)